docs/tutorials/vagrant-discovery.rst

```diff
@@ -0,0 +1,370 @@
+Automatic Discovery and Catalog Server Nodes
+===============================================
+
+In this module, you will learn about RackHD's discovery, catalog and poller functionality by using
+the simulated nodes that were set up in the previous modules.
```

Comment:
It seems previous section (Local Vagrant-based Environtment Set up) doesn't mention how to start vnode using vagrant file. I saw instruction of booting vnode the the following section of this file, so maybe this line here needs to be modified.